.github/workflows/test-ui-e2e-only-desktop.yml

```diff
@@ -109,9 +114,12 @@ jobs:
         env:
           INPUTS_TEST_FILTER: ${{ inputs.test_filter }}
           SEED: ${{ secrets.SEED_QAA_B2C }}
+          XRAY_CLIENT_ID: ${{ secrets.XRAY_CLIENT_ID }}
+          XRAY_CLIENT_SECRET: ${{ secrets.XRAY_CLIENT_SECRET }}
           ENABLE_BROADCAST: ${{ contains(inputs.enable_broadcast, 'true') }}
           SPECULOS_TESTS: ${{ !contains(inputs.speculos_tests, 'false') }}
-
+          XRAY: ${{ contains(inputs.export_to_xray, 'true') }}
```

Comment:
could be simplified to `${{ inputs.export_to_xray }}`

Reply:
done